modules/oe_content_event/config/install/field.field.node.oe_event.field_report_title.yml

```diff
@@ -0,0 +1,17 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_report_title
+    - node.type.oe_event
+id: node.oe_event.field_report_title
+field_name: field_report_title
+entity_type: node
+bundle: oe_event
+label: 'Report title'
+description: ''
+required: false
```

Comment:
Should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Report fields group filled in.

Reply:
@nagyad Missing validation for the report composite

Reply:
@upchuk Please see my comment in ticket (result of discussion with @kugta and @nagyad ).